Running Java benchmark MG, class D, on 2 servers (np=2), tcp,self

Servers:
mpi003
mpi032

mpirun --mca btl tcp,self --bind-to core --map-by node -np 2 java NPB_MPJ.MG class=D

Start: Mon Feb 17 10:33:22 PST 2014
===============================================================================
 Unofficial NAS Parallel Benchmarks Java version
 MPJ Version MG CLASS=D np=2
 No input file mg.input. Using compiled defaults
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 6014203
	at NPB_MPJ.MG.zero3(MG.java:639)
	at NPB_MPJ.MG.runBenchmark(MG.java:241)
	at NPB_MPJ.MG.main(MG.java:112)
-------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code.. Per user-direction, the job has been aborted.
-------------------------------------------------------
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 6014203
	at NPB_MPJ.MG.zero3(MG.java:639)
	at NPB_MPJ.MG.runBenchmark(MG.java:241)
	at NPB_MPJ.MG.main(MG.java:112)
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[52648,1],0]
  Exit code:    1
--------------------------------------------------------------------------
===============================================================================
End: Mon Feb 17 10:33:26 PST 2014
